frontend/src/metabase/components/CollapseSection.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint "react/prop-types": 2 */
+
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import Icon from "metabase/components/Icon";
+
+function CollapseSection({
+  children,
+  className,
+  header,
+  headerClass,
+  bodyClass,
+  initialState = "collapsed",
```

Comment:
It feels odd to have two strings represent what's essentially a boolean. `isInitiallyExpanded` maybe? That's pretty verbose though.

Reply:
In general, especially when it comes to interfaces, I try to defer to the usage of strings over booleans because they're more descriptive. That's not entirely true here as a `isInitiallyExpanded` prop is probably just as clear as `initialState="expanded"`, but the upside of using a string is that we can add additional states to this component in the future while avoiding adding additional props.